daemons/ipa-slapi-plugins/ipa-otp-lasttoken/ipa_otp_lasttoken.c

```diff
@@ -105,6 +105,41 @@ static bool sdn_is_only_enabled_token(Slapi_DN *target_sdn, const char *user_dn)
     return result;
 }
 
+static bool is_otp_enabled(const char *user_dn)
+{
+    char *attrs[] = { "ipaUserAuthType", NULL };
+    Slapi_Entry *entry = NULL;
+    Slapi_DN *sdn;
+    const Slapi_DN *base;
+    uint32_t authtypes;
+    int search_result = 0;
+    char *authConfigDN;
+
+    sdn = slapi_sdn_new_dn_byval(user_dn);
+    base = slapi_get_suffix_by_dn(sdn);
```

Comment:
I don't see a ``slapi_sdn_free()`` call for base. Does ``slapi_get_suffix_by_dn`` return a borrowed reference?